cmd/pb-td-gen/traverse_linked_rpc.go

```diff
@@ -0,0 +1,117 @@
+package main
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+)
+
+// AllowedClasses is ...
+type AllowedClasses string
+
+// defines classes on which it is allowed to get fields
+const (
+	LinkedRPC    AllowedClasses = "main.linkedRPC"
+	ProtoRPC                    = "proto.RPC"
+	ProtoMessage                = "proto.Message"
+	ProtoComment                = "proto.Comment"
+)
+
+// defines which fields are available for every specific class
+var allowedAccess = map[AllowedClasses][]string{
+	LinkedRPC: {"rpcFunction", "responseMsg", "requestMsg"},
+	ProtoRPC:  {"Position", "Comment", "Name"}, /*"RequestType", "StreamsRequest", "ReturnsType", "StreamsReturns",
+	"Elements", "InlineComment", "Parent"*/
+	ProtoComment: {"Position", "Lines"}, /*Cstyle, ExtraSlash*/
+	ProtoMessage: {"Position", "Comment", "Name" /*, "IsExtend", "Elements", "Parent"*/},
+}
+
+// traverseCallParts runs recursively through the nodes to extract a value for the given blueprintValueParts
+// @param object: the object is the actual node where the recursive function is working on in this iteration
+// @param blueprintValueParts: the slice where the path is stored to access a value
+// @param actPartsPos: the actual position in this slice
+func traverseCallParts(object reflect.Value, blueprintValueParts []string, actPartsPos int) string {
```

Comment:
Really? Do it absolutely need to be a reflect.Value...? I think it is quite unusual to pass reflect properties to a function.